jubilant/statustypes.py

```diff
@@ -254,6 +254,26 @@ def is_waiting(self) -> bool:
         """Report whether the application status for this app is "waiting"."""
         return self.app_status.current == 'waiting'
 
+    @property
+    def leader_unit(self) -> UnitStatus:
+        """Return the leader unit's status.
+
+        For example, instead of duplicating the application name::
+
+            address = status.apps['snappass-test'].units['snappass-test/0'].address
+
+        You can simplify to::
+
+            address = status.apps['snappass-test'].leader_unit.address
+
+        Raises:
+            ValueError: if this application has no units marked as leader.
```

Comment:
I feel like this should be:

```suggestion
            ValueError: if this application has no unit marked as leader.
```

Since there can only be 0 or 1 leaders.